README.md

```diff
@@ -1,5 +1,7 @@
 # earthdata-hashdiff
 
+[![Available on pypi](https://img.shields.io/pypi/v/earthdata-hashdiff.svg)](https://pypi.python.org/pypi/earthdata-hashdiff/)
```

Comment:
This should just add a badge to the README, when rendered in GitHub, that shows the version number of the latest release in PyPI. It is also clickable to go to the PyPI project page.